crates/dojo-lang/src/compiler.rs

```diff
@@ -157,6 +198,17 @@ impl Compiler for DojoCompiler {
     }
 }
 
+fn get_allowed_libfuncs_list(props: &Props, unit: &CairoCompilationUnit) -> ListSelector {
+    match &props.allowed_libfuncs_list {
+        Some(SerdeListSelector::Name { name }) => ListSelector::ListName(name.clone()),
+        Some(SerdeListSelector::Path { path }) => {
+            let path = path.relative_to_file(unit.main_component().package.manifest_path()).unwrap_or_default();
+            ListSelector::ListFile(path.into_string())
+        }
+        None => Default::default(),
+    }
+}
+
```

Comment:
```suggestion
```
Can be removed for now for the reason mentioned above.

Reply:
Sure @glihm all suggestions were applied. Thank so much!!